### PR TITLE
docs:image build and usage

### PR DESCRIPTION
**Pull request by lec-bit**
1、Added documentation for online compilation images, including usage, construction, and runtime process of the images. 
2、added some annotations to the yaml files.

### CODE REVIEW

File: README-zh.md [hunk 1]
```diff
@@ -83,6 +84,13 @@ Kmesh创新性的提出将流量治理下沉OS，在数据路径上无需经过
   
   默认使用Kmesh功能，可通过调整yaml文件中的启动参数进行功能选择
   
+  ```sh
+  [root@ docker]# vim kmesh.yaml 
+  ...
+             args: ["./start_kmesh.sh -enable-kmesh -enable-ads=true"]
```
Comment (hzxuzhonghu) [inline README-zh.md:90]: What's your intention here

File: docs/kmesh_compile-zh.md [hunk 2]
```diff
@@ -137,6 +137,62 @@ Kmesh需要在拥有Kmesh内核增强特性的Linux环境中编译构建。当
   kmesh-0.0.1           latest              e321b18d5fee        4 hours ago         675MB
   ```
 
+### docker image [兼容模式](../build/docker/README.md)构建
```
Comment (hzxuzhonghu) [inline docs/kmesh_compile-zh.md:140]: I donot see the difference with L92, can you explain

File: docs/kmesh_compile-zh.md [hunk 3]
```diff
@@ -137,6 +137,62 @@ Kmesh需要在拥有Kmesh内核增强特性的Linux环境中编译构建。当
...
+
+  查看本地镜像仓库已有Kmesh镜像
+
+  ```sh
+  [root@dev docker]# docker images
+  REPOSITORY            TAG                 IMAGE ID            CREATED             SIZE
+  kmesh                 v0.1.0              fd60c89b1253        4 days ago          1.5GB
+  ```
+
+- 镜像获取
```
Comment (hzxuzhonghu) [inline docs/kmesh_compile-zh.md:188]: I think we donot need this

File: README-zh.md (hunk 1 above)
Comment (lec-bit) [reply README-zh.md:90]: Here is an example of selecting a function by adjusting the startup parameters in the YAML file.

Comment (lec-bit) [reply README-zh.md:90]: Here is an example of selecting a function by adjusting the startup parameters in the YAML file.

File: docs/kmesh_compile-zh.md (hunk 2 above)
Comment (lec-bit) [reply docs/kmesh_compile-zh.md:140]: In L92, images are built in RPM mode. In this example, online compilation (images in compatible mode) is built using kmesh source code. The main difference is that rpm is replaced with kmesh source code. The start_kmesh.sh script contains adaptive compilation and installation actions. Other actions at the user construction level are the same.

File: build/docker/README.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
+[TOC]
+
+## docker镜像说明
+
+由于kemsh的部分能力对于内核版本有着较强的依赖，我们提供了两种kmesh的镜像：
+
+一种是可直接在指定环境上使用的镜像，该镜像可提供kemsh的七层网络流量治理；
```
Comment (nlgwcy) [inline build/docker/README.md:7]: kemsh -> kmesh

File: README-zh.md (hunk 1 above)
Comment (hzxuzhonghu) [reply README-zh.md:90]: This is not an automatical way, in future we can support customize and helm.

No need to provide this in readme

File: docs/kmesh_compile-zh.md (hunk 2 above)
Comment (hzxuzhonghu) [reply docs/kmesh_compile-zh.md:140]: I see both use the same dockerfiler `docker build -f kmesh.dockerfile -t kmesh-0.0.1 .`

File: docs/kmesh_compile-zh.md [hunk 5]
```diff
@@ -137,6 +137,62 @@ Kmesh需要在拥有Kmesh内核增强特性的Linux环境中编译构建。当
...
+    -rw-r--r--. 1 root root  764 Nov 25 01:31 start_kmesh.sh
+    ```
+
+    将镜像原料放到一个目录下
+
+    ```sh
+    [root@dev docker]# ll
+    -rw-r--r--. 1 root root  793 Nov 25 01:36 kmesh.dockerfile
+    -rw-r--r--. 1 root root  764 Nov 25 01:36 start_kmesh.sh
+    -rw-r--r--. 1 root root  764 Nov 25 01:36 xxx
```
Comment (hzxuzhonghu) [inline docs/kmesh_compile-zh.md:170]: can we provide a script doing this

File: build/docker/README.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+
+​		多数时候是由于宿主机上没有安装kernel-headers，导致部分头文件文件读取失败，无法正确识别宿主机环境，容易导致编译失败
+
+2、kmesh.ko编译失败：
+
+​		宿主机上没有安装kerne-devel软件包，导致缺少编译ko的依赖
+
+3、镜像启动时找不到/root/.kube/config 文件，启动失败：
+
+​		由于不同的K8S环境的创建方式不同，config位置也不同，用户需要在kmesh启动的时候去指定kmesh.yaml文件中的kube-config-path的hostpath，将其映射到镜像中的指定位置
```
Comment (hzxuzhonghu) [inline build/docker/README.md:84]: Maybe I miss some context, why do kmesh need kubeconfig path?

File: README-zh.md (hunk 1 above)
Comment (lec-bit) [reply README-zh.md]: OK，Updated

File: docs/kmesh_compile-zh.md (hunk 3 above)
Comment (lec-bit) [reply docs/kmesh_compile-zh.md]: This section has been refreshed

File: build/docker/README.md (hunk 4 above)
Comment (lec-bit) [reply build/docker/README.md:7]: refreshed

File: docs/kmesh_compile-zh.md (hunk 5 above)
Comment (lec-bit) [reply docs/kmesh_compile-zh.md:170]: Do you mean that a script is provided to automatically build a kmesh image? In the future, we can invoke the script on the access control CI to build an image？

File: build/docker/README.md (hunk 6 above)
Comment (hzxuzhonghu) [reply build/docker/README.md:84]: We will remove this soon.

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

We can update in future when we provide the build tool image

**Pull request merged**